src/core/File.ts

```diff
@@ -590,11 +592,7 @@ export class File {
             let jsonSourceMaps = JSON.parse(this.sourceMap);
             jsonSourceMaps.file = this.info.fuseBoxPath;
             jsonSourceMaps.sources = jsonSourceMaps.sources.map((source : string) => {
-                if(source.indexOf('stdin') !== -1) {
```

Comment:
Anything we can do about stdin? Why did you remove it?

Reply:
`stdin` shouldn't be there anymore because `file` is additionally provided to node-sass options - we will have the right file name with relativ path instead `stdin`. See SassPlugin comment.